digiroad2-api-viite/src/main/scala/fi/liikennevirasto/digiroad2/SearchApi.scala

```diff
@@ -32,14 +43,19 @@ class SearchApi(roadAddressService: RoadAddressService,
   private val getRoadAddress: SwaggerSupportSyntax.OperationBuilder =
     (apiOperation[List[Map[String, Any]]]("getRoadAddress")
       .parameters(
-        queryParam[String]("linkId").description("LinkId of a road address"),
-        queryParam[Double]("startMeasure").description("startMeasure of a road address").optional,
-        queryParam[Double]("endMeasure").description("endMeasure of a road address").optional
-      )
-      tags "SearchAPI (oth)"
-      summary "Gets all the road addresses in between the given linear location."
+        headerParam[String]("X-API-Key").description(XApiKeyDescription),
+        queryParam[String]("linkId").required.description("(KGV) LinkId for the link whose addresses are be returned"),
+        queryParam[Double]("startMeasure").optional.description("(Optional) Linear locations, whose endMeasure is before this <i>startMeasure</i>, are not returned."),
+        queryParam[Double]("endMeasure").optional.description("(Optional) Linear locations, whose startMeasure is after this <i>endMeasure</i>, are not returned.")
       )
+      tags "SearchAPI (Digiroad)"
+      summary "Returns all the road addresses for the given single link. Return values are listed as linear locations. Linear locations can optionally be restricted by the link's measure values."
+      description "Returns the road addresses of the given link, listed as linear locations." +
+                  "Linear locations may be restricted by giving <i>startMeasure</i>, and/or <i>endMeasure</i>. " +
+                  "A linear location must belong to the measure interval at least in one point, to be included in the returned results."
+    )
 
+  // TODO: "?" in the end is useless; does not take query params
```

Comment:
hmmm? Mitäs nuo linkId, startMeasure ja endMeasure ovat jos ei query parametreja? 🤔

Reply:
My bad, niitähän ne 😅 
Copy-paste väärään funktioon, väitän (seliseli)
Fiksattu